docs/drafts/BEP-0001.md

```diff
@@ -41,14 +41,17 @@ Adding a populationAbundance() function would allow capturing population level c
  
 ## Discussion
 
-* Should we use this for cell populations as well?
+* Should we use this for cell populations as well? E.g., "T-helper 17 cell"?
+* Similar to other BEL abundance types, should the loc() modifier apply? E.g., pop(TAX:"Demodex folliculorum", loc(UBERON:"hair follicle"))
+* Almost all other BEL abundance types can participate in complexes. Should pop() participate in complexes? Or do we need a new entity or relationship to represent physical interactions between populations and/or populations and other abundances?
+* How should symbiotic or parasitic relationships be represented?
```

Comment:
Can you give an english language example of reasoning over symbiotic or parasitic relationships? I'm not really familiar with how people talk about this scientifically.

Reply:
This is really meant to be a question, related to the complex() question above it. I am going to move it to the same bullet so that is more clear. I'd like to lay out consistent rules for functions across abundances, but am struggling with the idea of a complex composed of, for example, a deer and a tick.